### PR TITLE
[#198 ]Laguage translation with AOT (connects #198)

### PR DESCRIPTION
**Pull request by Pr0chin**
nepali1
nepali

Languages Translation works but it runs with different port ** ng serve --aot --i18nFile=src/locale/messages.fr.xlf --i18nFormat=xlf --locale=fr ** giving specific path.

### CODE REVIEW

**Comment by Pr0chin** [comment]
* But arabic it dosen't show any translation. *
* messages..xlf  '/ ' has been changed in to '\' or vice versa as i m running Linux ubuntu in my machine . *

File: src/main.ts [hunk 1]
```diff
@@ -8,4 +8,17 @@ if (environment.production) {
   enableProdMode();
 }
 
-platformBrowserDynamic().bootstrapModule(AppModule);
+// use the require method provided by webpack
+declare const require;
+// we use the webpack raw-loader to return the content as a string
+
+const translations = require(`raw-loader!./i18n/messages.ne.xlf`);
```
Comment (lmmrssa) [inline src/main.ts]: why are these hardcoded

**Comment by lmmrssa** [comment]
why there are src/messages and src/i18n/messages

**Comment by Pr0chin** [comment]
ok i ll remove  src/messages .

File: src/main.ts (hunk 1 above)
Comment (Pr0chin) [reply src/main.ts]: just to check it works or not as we can make dynamic  later on as per required language translation.

**Comment by Pr0chin** [comment]
@razu9861 i jst put message file in src folder as it is already exists in i18n folder. so I removed that folder.

File: src/main.ts (hunk 1 above)
Comment (lmmrssa) [reply src/main.ts]: all these are served when you load ng serve with locale mentioned as AOT has defined so why do we need to mention all files and locale here

File: src/i18n/messages.xlf [hunk 2]
```diff
@@ -6,15 +6,15 @@
         <source>
                     Oops!</source>
         <context-group purpose="location">
-          <context context-type="sourcefile">app\page-not-found\page-not-found.component.ts</context>
+          <context context-type="sourcefile">app/page-not-found/page-not-found.component.ts</context>
```
Comment (paulbert) [inline src/i18n/messages.xlf]: I'm also curious why this PR has changed all of the `\` to `/`...

**Comment by lmmrssa** [comment]
@paulbert  could you look after #151 
It's not supposed to be there but to test xlif file from crowdin which only works with master and l10n_master branch, changes has been tested on this old PR. Once you feel changes are good we will create separate PR for it.

**Pull request closed without merging**